openwisp_notifications/tasks.py

```diff
@@ -25,3 +27,15 @@ def delete_obsolete_notifications(instance_app_label, instance_model, instance_i
         | Q(target_object_id=instance_id)
     )
     Notification.objects.filter(where).delete()
+
+
+@shared_task
+def delete_old_notifications(days):
```

Comment:
Please don'r add days as argument to this this task, otherwise user will have to tweak celery beat's configuration for changing number of days which is not as easy for user. Instead we can provide a setting, say `OPENWISP_NOTIFICATIONS_REMOVE_OLDER`. The user will only have to assign a value to this setting in `settings.py` of their project. 

Set `90 ` as default for this setting. 

Use the configured value of this setting in your shared task. 

Assigning `None` to this setting should not delete notifications. 

Look how other settings works in [openwisp_notifications/settings.py](https://github.com/openwisp/openwisp-notifications/blob/master/openwisp_notifications/settings.py).